.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      python_version: python3.7
```

Comment:
Why bring in another requirement vs using git hooks?

Reply:
I think I forgot to remove this when experimenting with git-hooks. wanted to run black on pre-commit but since we use git external of the container this wouldn't work for everyone.